mlf_core/sync/sync.py

```diff
@@ -159,14 +168,19 @@ def make_template_project(self):
         Delete all files and make a fresh template.
         """
         print('[bold blue]Creating a new template project.')
-        # dry create run from mlf-core in tmp directory
+        # dry create run from dot_mlf_core in tmp directory
         with tempfile.TemporaryDirectory() as tmpdirname:
-            # COOKIETEMPLE TODO: Refactor this by passing a path param to the create and choose_domain functions, which default to PWD if not passed
             old_cwd = str(Path.cwd())
+            log.debug(f'Saving current working directory {old_cwd}.')
             os.chdir(tmpdirname)
+            log.debug(f'Changed directory to {tmpdirname}.')
+            log.debug(f'Calling choose_domain with {self.dot_mlf_core}.')
+            # choose_domain(path=Path.cwd(), domain=None, dot_cookietemple=self.dot_cookietemple)
```

Comment:
This line must be removed, right?

> choose_domain(domain=None, dot_mlf_core=self.dot_mlf_core)

is below

Reply:
Yeah, my bad. Was just a reminder that I have to fix this here as well as soon as I add the create output path parameter